config/iothub/config.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package iothub
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+
+	"github.com/crossplane-contrib/provider-jet-azure/apis/rconfig"
+	"github.com/crossplane-contrib/provider-jet-azure/config/common"
+)
+
+const (
+	groupIOTHub = "iothub"
+)
+
+// Configure configures iothub group
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("azurerm_iothub", func(r *config.Resource) {
+		r.Kind = "IOTHub"
+		r.ShortGroup = groupIOTHub
```

Comment:
nit: we might consider overriding default group for all iothub resources in default resource, like: https://github.com/crossplane-contrib/provider-jet-gcp/blob/ca147c40ebe349cb933396624890bc9b92745da8/config/overrides.go#L14

Reply:
Thanks @turkenh! Done.